src/containers/AccessPointDetails/components/General/index.js

```diff
@@ -0,0 +1,1017 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Card, Form, Input, Table, Collapse, Select, Tooltip } from 'antd';
+import { InfoCircleOutlined } from '@ant-design/icons';
+import Button from 'components/Button';
+import styles from '../index.module.scss';
+
+const General = ({ data }) => {
+  const { Item } = Form;
+  const { Option } = Select;
+  const { Panel } = Collapse;
+
+  const [form] = Form.useForm();
+
+  const layout = {
+    labelCol: { span: 5 },
+    wrapperCol: { span: 10 },
+  };
+
+  const handleOnSave = () => {
+    form
+      .validateFields()
+      .then(values => {
+        console.log(values);
+      })
+      .catch(() => {});
+  };
+
+  form.setFieldsValue({
```

Comment:
this should be set in a useEffect or it will be called on every re-render
`useEffect(() => {
    form.setFieldsValue({ ... });
  }, []);`